### ISSUE TITLE
ReddCoin Core Wallet does not open

### ISSUE THREAD

**Issue by roleerob** [reddcoin-project/reddcoin-3.10#138]
After successfully using your wallet for weeks, starting yesterday it no longer opens.  I have captured the details in the attached PDF.

Operating System:  Windows 10, 64-bit

Please help me resolve this problem.  Thank you!

roleerob

[ReddCoin Core Startup Errors.pdf](https://github.com/reddcoin-project/reddcoin/files/2979208/ReddCoin.Core.Startup.Errors.pdf)

**Comment by reddink** [comment]
First question is always what has changed, in either configuration or the way you launch Reddcoin
But a couple of things to check,
make sure Reddcoin is not running in the background (use task manger to make sure there are no Reddcoin processes running)
AntiVirus is not impacting operation (add to exception list)
can you supply log files and reddcoin.conf

**Comment by roleerob** [comment]
Thank you for writing back.  In response:

1.  Nothing has changed that I am aware of - not in launching, not in configuration, not in anything.  For example, I would assume if it was something substantial in Windows, I would see other problems on my computer.  I don't have any other problems, of which I am aware.

2.  When I have Task Manager open, then launch ReddCoin, it says power use is Very High, then disappears, with the icon still on the taskbar, as described in my original message.  That is, nothing has changed and there is no Reddcoin process running.

3.  I use PC Matic and their Super Shield product.  Following the steps to add ReddCoin to their whitelist doesn't show anything, i.e. on launching ReddCoin Core it does not ever show up to add it ...

4.  There is no file on my hard drive named reddcoin.conf.  As for log files, there are several in the subfolders under ...\AppData\Roaming\ReddCoin.  You want all of them which end in *.log?  Until I know for sure what you want, I have uploaded debug.log

[debug.log](https://github.com/reddcoin-project/reddcoin/files/2994028/debug.log)

**Comment by reddink** [comment]
Thanks for the debug.log

I have taken a first pass at the file, and operationally, the wallet seems to behaving correctly.
Out of all the start/stops that have occurred, there were only 2 abnormal shutdowns (abrupt terminations)
All the others, the wallet terminated because it was passed a termination signal

When the application has been running for a period of time, i can see that, it is waiting to be unlocked (good to see you have encrypted)

At this point I am going to suspect an update to one of your applications (antivirus probably) has started monitoring, and is terminating
I am not familiar with PC matic.
I would suggest to check their logs to see if it is impacting, alternatively temporarily disable to verify operations

The other thing that *may* have occurred is the coordinates for the application position have pushed off the viewable workspace (do you run multiple monitors??)
you could check the following 
HKCU\Software\Reddcoin\Reddcoin-Qt\nWindowPos
HKCU\Software\Reddcoin\Reddcoin-Qt\nWindowSize


 
2019-03-09 23:42:05 wallet completed syncing.
Wallet was in a locked state until 
2019-03-10 00:07:16 Requesting shutdown (normal shutdown)

2019-03-12 17:22:29 Requesting initialize (starting)
2019-03-12 17:24:10 wallet completed syncing
2019-03-13 02:05:47 Abrupt termination (wallet not shutdown) ???

2019-03-13 03:48:01 Requesting initialize (starting)
2019-03-13 03:48:43 wallet completed syncing
...
2019-03-13 17:32:37 wallet appears to loose network connectivity
2019-03-13 17:33:41 network resumes
2019-03-14 17:18:56 Requesting shutdown (normal shutdown)

2019-03-14 18:53:15 Requesting initialize (starting)
2019-03-14 18:54:07 wallet completed syncing
...
a block staked.
...
2019-03-15 11:18:54 Abrupt termination (wallet not shutdown) ???

2019-03-15 11:24:47 Requesting initialize (starting)
2019-03-15 11:25:31 wallet completed syncing
2019-03-16 17:55:19 Requesting shutdown (normal shutdown)

2019-03-16 20:16:55 Requesting initialize (starting)
2019-03-16 20:17:41 wallet completed syncing
2019-03-17 03:23:11 wallet appears to loose network connectivity
2019-03-17 03:24:37 network resumes
2019-03-17 19:53:49 Requesting shutdown (normal shutdown)

2019-03-18 01:19:32 Requesting initialize (starting)
2019-03-18 01:20:17 wallet completed syncing
2019-03-18 01:22:20 Requesting shutdown (normal shutdown)

2019-03-18 01:22:58 Requesting initialize (starting)
2019-03-18 01:23:54 Requesting shutdown (normal shutdown)

2019-03-18 01:33:50 Requesting initialize (starting)
2019-03-18 01:35:26 Requesting shutdown (normal shutdown)

2019-03-18 01:35:57 Requesting initialize (starting)
2019-03-18 01:37:00 Requesting shutdown (normal shutdown)

2019-03-18 14:21:40 Requesting initialize (starting)
2019-03-18 14:22:55 wallet completed syncing
2019-03-18 14:30:36 Requesting shutdown (normal shutdown)

2019-03-18 14:57:54 Requesting initialize (starting)
2019-03-18 14:58:55 Requesting shutdown (normal shutdown)

2019-03-18 15:03:00 Requesting initialize (starting)
2019-03-18 15:08:26 Initialization result: 0 << tried to run a second instance
2019-03-18 15:46:08 Requesting shutdown (normal shutdown)

2019-03-18 15:52:14 Requesting initialize (starting)
2019-03-18 16:15:10 Requesting shutdown (normal shutdown)

2019-03-18 16:23:46 Requesting initialize (starting)
2019-03-18 16:25:21 Initialization result: 0 << tried to run a second instance
2019-03-18 16:37:28 Requesting shutdown (normal shutdown)

2019-03-21 20:43:53 Requesting initialize (starting)
2019-03-21 20:45:18 Still syncing
2019-03-21 20:45:18 Requesting shutdown (normal shutdown)

2019-03-21 21:05:21 Requesting initialize (starting)
2019-03-21 21:06:28 wallet completed syncing
2019-03-21 21:06:31 Initialization result: 0 << tried to run a second instance
2019-03-21 21:06:50 Requesting shutdown (normal shutdown)

2019-03-21 21:10:17 Requesting initialize (starting)
2019-03-21 21:10:47 wallet completed syncing
2019-03-21 21:24:10 Requesting shutdown (normal shutdown)

**Comment by roleerob** [comment]
Thank you for your response Reddink!  In response:

1.  I have already tried to stopping the PC Matic security checks.  This is a fairly common recommendation, for example, when installing new software.  It is not hard to do.  But ...  It has no affect on my problem.

2.  I do, in fact, run multiple monitors - three of them.  Two standard monitors, plus my laptop screen.  That said, I do not know to what this is referring:
HKCU\Software\Reddcoin\Reddcoin-Qt\nWindowPos
HKCU\Software\Reddcoin\Reddcoin-Qt\nWindowSize
Where would I go / what would I do with this info?

3.  What I have chosen NOT to do, up until now is simply reinstall the wallet.  Lots of questions about that, which I preferred to only leave as a last option.  I do NOT want to disrupt all that I have done in staking, etc.  I DO have the wallet backed up.  So ...  Would I first uninstall, then reinstall?  Or just try reinstalling?  If I do reinstall, what will I be facing, when trying to get the wallet running smoothly again?  I have noted, so far at least, you have not made that recommendation yourself.

Thanks again for your help on this!

**Comment by roleerob** [comment]
It has been nearly two weeks since I heard from you.  Any update on my last message?

**Comment by reddink** [comment]
Hi

HKCU\Software\Reddcoin\Reddcoin-Qt\nWindowPos
HKCU\Software\Reddcoin\Reddcoin-Qt\nWindowSize
are 2 registry settings that save the screen position the wallet was last running.
I would be curious to know what was in those settings

from windows run command
regedit
navigate to Computer/HKEY_CURRENT_USER\Software\Reddcoin\Reddcoin-Qt\
amd let me know the values of those 2 keys

Alternatively, 
Uninstalling/reinstalling would also require downloading the blockchain again
if you have a backup of your wallet.dat file, and know where to restore it, that would be a standby option

In summary, from the log file, it appears to be running
And my guess is that the wallet is off screen
(by the way, after you start the wallet, do you see the icon in the task bar?)

**Comment by roleerob** [comment]
Good morning (here).  Thank you for your response.  With your directions, I found the two values you seek and have just copied and pasted the image of them here, to avoid getting it wrong:

![image](https://user-images.githubusercontent.com/31572169/55731755-f680fd00-59d7-11e9-9d1e-c5d358703266.png)

As for whether I see the icon in the task bar, that is covered in the PDF attached to my original message.  Everything in that PDF is unchanged right now, including that, upon launching, the icon does appear on the task bar ...  And still getting the same error messages ...

And, just in case, I rechecked the two register values you seek.  They are also unchanged, now that I have rerun ReddCoin Core again this morning.

As far as reinstalling, I do have the wallet.dat file backed up from Mar 14th.  My problems began a day or two later.  I tried just overwriting the wallet.dat file with this backed up copy on Mar 14th, but it had no effect.  You response on that option says *"know where to restore it"*, so are you referring to which directory to place it in? 

So ...  Nothing has changed ...  Please advise on what these register values indicate to you.  Or any other ideas you have for getting my wallet back in operation.

Thank you!

**Comment by lionzeye** [comment]
The current value in nWindowPos indicates that the wallet is off-screen, as @reddink mentioned.
Delete the nWindowPos register key and start the Reddcoin wallet.

**Comment by roleerob** [comment]
Okay, good news!  Happy to say @lionzeye that did the trick.  For the first time since Mar 17th, I am once again staking in my Reddcoin Core wallet.

Two thoughts, before closing this issue:

1.  Now that you know what my problem was, do you have any way of determining the root cause of the problem?  I'll answer any questions you ask, to the best of my ability.  To help you with troubleshooting for whatever that may do to help improve the wallet code.  For starters, I have two full-size monitors, plus my laptop screen active at all times.

2.  Can we agree the error / status? messages sent with my original PDF didn't help much?  If you have some way of improving the error trapping message for us, that would be great.  I don't claim to know anything about it, but both of those messages don't give any indication there is actually a problem with the wallet being "off screen" (still not sure what that even means ...).

Thanks for your help getting me back up and running, without having to reinstall!

**Comment by TechAdeptRDD** [comment]
https://www.reddcoin.com/2020/06/13/reddcoin-core-3-10-0/

**Comment by roleerob** [comment]
Noting you had closed this issue @TechAdeptRDD, I investigated and found the upgrade you had published.  And installed it and ran it continuously, as I have done for months now.  For a time, it appeared to work fine.

But ...

Yesterday, I was once again faced with it not opening.  Thankfully, as detailed earlier in this thread, I knew what has worked in the past to resolve the problem.  I must open RegEdit and delete the nWindowPos register key there.  Then I can restart Reddcoin wallet and it works fine.

So ...

This is a nuisance and it is still a problem unresolved.  Thankfully, I only experience it every few weeks or so.  And I know how to get it working again ...

Thought you would want to know.

Respectfully,

roleerob

**Comment by reddink** [comment]
Thanks Rob, will look into getting this issue resolved in next build

**Comment by hasanora** [comment]
I having a problem with the wallet. I using the latest wallet. I was working fine. Since yesterday I am noticing this issue.  After launching the wallet I am not able to view the wallet in the desktop.  However, I checked the task manager that showed the process is running. I rebooted that machine multiple times. Please advice if there is any solution.
Running on win10 pro
 
![reddcoin](https://user-images.githubusercontent.com/13524004/92133250-26535080-edd6-11ea-9e63-dd953fbc4435.png)
